cmd/botkube/main.go

```diff
@@ -136,8 +137,14 @@ func run(ctx context.Context) error {
 	}
 	defer pluginManager.Shutdown()
 
+	logger.Info("Found kube config: ", conf.Settings.Kubeconfig)
+
+	cfgLoader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
```

Comment:
Use config loader explicitly - this will allow access to both, [RawConfig](https://github.com/kubernetes/client-go/blob/1517ffb8d37c99e6a3a2842bcdee0aa271f0332b/tools/clientcmd/client_config.go#L128-L130) (entire config) and [ClientConfig](https://github.com/kubernetes/client-go/blob/1517ffb8d37c99e6a3a2842bcdee0aa271f0332b/tools/clientcmd/client_config.go#L133) (sub-set of raw config).